### PR TITLE
Resolve minor readme bug due to API change

### PR DESCRIPTION
**Pull request by eedeleon**
## Types of changes
Minor readme update. 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Docs change / refactoring / dependency upgrade

## Motivation and Context / Related issue

While trying to get a sample working I noticed:
class PrivacyEngine:
    def __init__(
        self,
        module: nn.Module,
        batch_size: int,
        sample_size: int,
        alphas: List[float],
        noise_multiplier: float,
        max_grad_norm: float,
        grad_norm_type: int = 2,

However the README.md had:
privacy_engine = PrivacyEngine(
    model,
    train_loader,
    alphas=[1, 10, 100],
    noise_multiplier=1.3,
    max_grad_norm=1.0)

## How Has This Been Tested (if it applies)

My code did not fail. After I deleted the Batchnorm layers of the model but that was a separate issue that I would like to understand more but maybe later.

# Use pytorch-dp to privatize training
``` code-block:: python
from torchdp import PrivacyEngine
privacy_engine = PrivacyEngine(
    model,
    BATCH_SIZE,
    1,
    alphas=[1, 10, 100],
    noise_multiplier=1.3,
    max_grad_norm=1.0,
)

privacy_engine.attach(optimizer)
```
## Checklist

- [X] The documentation is up-to-date with the changes I made.
- [X] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [ X All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Comment by JohnlNguyen** [comment]
Thank you for updating doc. I will merge this request shortly
